docs/en/pact-functions.md

```diff
@@ -204,9 +204,9 @@ pact> (format "My {} has {}" ["dog" "fleas"])
 Compute BLAKE2b 512-bit hash of VALUE. Strings are converted directly while other values are converted using their JSON representation.
```

Comment:
```suggestion
Compute BLAKE2b 256-bit hash of VALUE. Strings are converted directly while other values are converted using their JSON representation.
```

Reply:
@larskuhtz these are auto-generated by the function definition in Native.hs, will implement suggestion there.